sentimentVisualizer/index.html

```diff
@@ -48,7 +50,15 @@
                 <div class="col-md-9" style="text-align:center;">
                     <form class="col-lg-12 hidden-xs ng-pristine ng-valid" action="/sentimentVisualizer/visualize.html">
                         <div class="input-group input-group-lg col-sm-8" style="float:left">
-                            <span class="input-group-btn"><button class="btn btn-lg btn-primary" ng-click=''>Visualize Emotion</button></span>
+<<<<<<< HEAD
+                            <span class="input-group-btn">
+                            <button class="btn btn-lg btn-primary" ng-click=''>
+                            Visualize Emotion
+                            </button>
+                            </span>
+=======
+                            <span class="input-group-btn"><button class="btn btn-lg btn-primary" ng-click="">Visualize Emotion</button></span>
```

Comment:
Line is longer than allowed. (142 > 129)

*LineLengthBear, severity NORMAL, section `default`.*